### PR TITLE
use alpine base image

### PR DESCRIPTION
**Pull request by yutongp**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
-FROM golang:1.14.4-stretch as build
+FROM golang:1.14.4-alpine as build
```
Comment (raullenchai) [inline Dockerfile:1]: https://hub.docker.com/_/golang shows 1.14.4-alpine doesn't exist, 14.6?

Comment (yutongp) [reply Dockerfile:1]: https://hub.docker.com/layers/golang/library/golang/1.14.4-alpine/images/sha256-2de21f4fe5c3557c31b716291a7986c0d54abbd282cd7fb43aaa484672b0c855?context=explore

File: Dockerfile [hunk 2]
```diff
@@ -14,7 +14,9 @@ COPY . .
 RUN mkdir -p $GOPATH/pkg/linux_amd64/github.com/iotexproject/ && \
     make clean build-all
 
-FROM debian:stretch
+FROM alpine:latest
+
+RUN apk add --no-cache ca-certificates
```
Comment (dustinxie) [inline Dockerfile:19]: this cannot be combined with above?

Comment (yutongp) [reply Dockerfile:19]: https://docs.docker.com/develop/develop-images/multistage-build/

**Pull request merged**